File: exam-system-java/server/src/main/java/com/yyjy/exam/question/bo/QuestionExcelTemplateBo.java
package com.yyjy.exam.question.bo;

import com.alibaba.excel.annotation.ExcelProperty;
import lombok.Builder;
import lombok.Data;

@Builder
@Data
public class QuestionExcelTemplateBo {

    @ExcelProperty("题目内容")
    private String content;

    @ExcelProperty("题目类型")
    private String type;

    @ExcelProperty("是否多选")
    private String multiple;

    @ExcelProperty("分类id")
    private String categoryId;

    @ExcelProperty("难度")
    private String difficulty;

    @ExcelProperty("分值")
    private String score;

    @ExcelProperty("选项A")
    private String choiceA;

    @ExcelProperty("选项B")
    private String choiceB;

    @ExcelProperty("选项C")
    private String choiceC;

    @ExcelProperty("选项D")
    private String choiceD;

    @ExcelProperty("答案")
    private String answer;

    @ExcelProperty("解析")
    private String analysis;
}

File: exam-system-server/src/main/kotlin/com/yyjy/models/bo/QuestionExcelTemplateBo.kt
package com.yyjy.models.bo

import com.alibaba.excel.annotation.ExcelProperty

/**
 * @author Nyxcirea
 * @date 2026/4/11
 * @description: TODO
 */
data class QuestionExcelTemplateBo(
    @ExcelProperty("题目内容")
    var content: String? = null,
    @ExcelProperty("题目类型")
    var type: String? = null,
    @ExcelProperty("是否多选")
    var multiple: String? = null,
    @ExcelProperty("分类id")
    var categoryId: String? = null,
    @ExcelProperty("难度")
    var difficulty: String? = null,
    @ExcelProperty("分值")
    var score: String? = null,
    @ExcelProperty("选项A")
    var choiceA: String? = null,
    @ExcelProperty("选项B")
    var choiceB: String? = null,
    @ExcelProperty("选项C")
    var choiceC: String? = null,
    @ExcelProperty("选项D")
    var choiceD: String? = null,
    @ExcelProperty("答案")
    var answer: String? = null,
    @ExcelProperty("解析")
    var analysis: String? = null
)
